src/Mutant/Calculator.php

```diff
@@ -137,7 +149,11 @@ public function getCoveredCodeMutationScoreIndicator(): float
 
         $score = 0.;
         $testedTotal = $this->totalCount - $this->notTestedCount;
-        $coveredTotal = $this->killedCount + $this->timedOutCount + $this->errorCount;
+        $coveredTotal = $this->killedCount + $this->errorCount;
+
+        if (!$this->treatTimeoutsAsEscapes) {
+            $coveredTotal += $this->timedOutCount;
+        }
```

Comment:
I had some time to think and now I think this should be the default. I'll write it out later.